setup/setup.sh

```diff
@@ -7,6 +7,12 @@ then
     PRESERVECONFIG=1
 fi
 
+ISACTIVE=`systemctl is-active traccar.service`
+if [ "${ISACTIVE}" == "active" ]
+then
+    systemctl stop traccar.service
+fi
+
 mkdir -p /opt/traccar
 cp -r * /opt/traccar
```

Comment:
Maybe we should add `-f` here as well?

Reply:
According to the man page '-f' will not work for the stop command:

```
       -f, --force
           When used with enable, overwrite any existing conflicting symlinks.

           When used with edit, create all of the specified units which do not already exist.

           When used with halt, poweroff, reboot or kexec, execute the selected operation without shutting down all units. However, all processes will be killed forcibly and all file systems are unmounted or
           remounted read-only. This is hence a drastic but relatively safe option to request an immediate reboot. If --force is specified twice for these operations (with the exception of kexec), they will be
           executed immediately, without terminating any processes or unmounting any file systems. Warning: specifying --force twice with any of these operations might result in data loss. Note that when
           --force is specified twice the selected operation is executed by systemctl itself, and the system manager is not contacted. This means the command should succeed even when the system manager has
           crashed.
```
There are two solutions:

1. Keep this as is. I never had problems that the traccar service never stopped.
2. After the stop check if the process are still running and then bail and throw an error so the user can solve it.

My preference would be option 1.

Reply:
I meant for the `cp` command.